src/components/Projects/ProjectBlock.tsx

```diff
@@ -0,0 +1,124 @@
+import { BodyText, HeaderText, PrimaryAccentText } from 'components/Text'
+import { JSX } from 'preact/jsx-runtime'
+import { useRef } from 'preact/hooks'
+import Button from 'components/Button'
+import Tilt from 'react-parallax-tilt'
+import classnames, {
+  TTextColor,
+  alignItems,
+  display,
+  dropShadow,
+  flexDirection,
+  gap,
+  inset,
+  justifyContent,
+  margin,
+  position,
+  rotate,
+  scale,
+  space,
+  width,
+  zIndex,
+} from 'classnames/tailwind'
+
+type Element = JSX.Element
+
+const wrapper = (reverse?: boolean, centered?: boolean) =>
+  classnames(
+    position('relative'),
+    display('flex'),
+    gap('gap-8'),
+    flexDirection(
+      { 'lg:flex-row-reverse': reverse, 'flex-col-reverse': reverse },
+      'flex-col',
+      'lg:flex-row'
+    ),
+    alignItems('items-center'),
+    justifyContent(centered ? 'justify-center' : 'justify-evenly'),
+    width('w-auto', 'md:w-full'),
+    margin('mx-4'),
+    zIndex('z-10')
+  )
+
+const descriptionBlock = classnames(
+  display('flex'),
+  flexDirection('flex-col'),
+  width('w-fit', 'sm:w-96'),
+  alignItems('items-start')
+)
+const cardHeader = space('space-y-4')
+const cardBody = space('space-y-6')
+
+const tiltWrapper = classnames(
+  scale('scale-50', 'tiny:scale-65', 'sm:!scale-100'),
+  position('absolute', 'sm:static'),
+  rotate('-rotate-20', 'sm:rotate-0'),
+  inset('-right-36', 'tiny:-right-28', 'bottom-80', 'tiny:bottom-64')
+)
+
+export default function ({
+  logo,
+  titleText,
+  titleColor,
+  subtitle,
+  description,
+  buttonTitle,
+  buttonUrl,
+  imageElement,
+  imageSource,
+  wrapReverse,
+  customGlareBorderRadius,
+  centered,
+}: {
+  logo?: Element
+  titleColor: TTextColor
```

Comment:
I don't like the idea of passing `TTextColor` also as the name `titleColor`. Can we change it to `color`, for instance?

Reply:
But `color` for what? We have subtitle and description here, simple `color` won't tell the dev what is this, the guy/girl will check the inner of the component to understand it

Reply:
But yes, we should limit the amount of props here